transcription/transcription.go

```diff
@@ -29,3 +30,15 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// ConvertAudioIntoRequiredFormat converts encoded audio into the required format.
+func ConvertAudioIntoRequiredFormat(fn string) error {
```

Comment:
What do you mean a non-static method? Aren't `Command` and `Run` static methods?

The format to use for mocking would be

```go
<package>.EXPECT().<function>(<argument1>, ..., ).Times(<number of expected calls>)
```
And you can optionally add a `Return` at the end if it must return something.